Trivia/plugin.py

```diff
@@ -301,6 +320,30 @@ def start(self, irc, msg, args, channel, num):
         irc.noReply()
     start = wrap(start, ['channel', optional('positiveInt')])
 
+    @internationalizeDocstring
+    def hint(self, irc, msg, args, channel):
+        """
+        
+        Invokes the next hint for the current question"""
+        channel = ircutils.toLower(channel)
+        if channel in self.games:
+            self.games[channel].hintcommand()
+        else:    
+            irc.reply("Trivia is currently not active in this channel.")
```

Comment:
Could you wrap the string in `_()` like the other commands? (it makes them translatable)